File: programme/volume-of-cylinder/volume-of-cylinder.cpp
// Write a program to find the Volume of a Cylinder

#include <iostream>
#include <math.h>

using namespace std;

int main()
{
    float radius, height, volume;

    cout << "Enter Radius of Cylinder : ";
    cin >> radius;
    cout << "Enter Height of Cylinder : ";
    cin >> height;

    // value of PI is defined in M_PI in math.h
    volume = M_PI * radius * radius * height;

    cout << endl
         << "Volume of Cylinder is : " << volume << endl;

    return 0;
}

File: programme/volume-of-cylinder/volume-of-cylinder.py
# Write a program to compute the volume of a Cylinder

import math

'''
    Author: dylan-power
    Date: 03/05/2022
'''

# Get radius and height from user
enter_radius = int(input('Radius : '))
enter_height = int(input('Height : '))

# Calculate volume
volume = enter_radius * enter_radius * enter_height * math.pi

# Print result
print("\nVolume : {:0.2f}".format((volume)))
